site/content/events/2016-kiel/_sponsors.txt

```diff
@@ -16,29 +16,41 @@ end
 <%= make_sponsor_links(@gsponsors) %>
 <br />
 <% if @gsponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a>
-<% elsif @gsponsors.size < 11 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a><% end %>
+<% elsif @gsponsors.size < 6 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Gold Sponsor!</a><% end %>
 
 <br/>
 <br/>
-<h1>Silver sponsors</h1>
+<h1>Host sponsor</h1>
 
-<% @ssponsors = [
+<% @hsponsors = [
+{ :image => "kiwi.png", :name => "Kieler Wirtschaftsförderung GmbH", :link => "http://www.kiwi-kiel.de/"},
 ] %>
-<%= make_sponsor_links(@ssponsors) %>
+<%= make_sponsor_links(@hsponsors) %>
 <br />
-<% if @ssponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a>
-<% elsif @ssponsors.size < 6 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Silver Sponsor!</a><% end %>
+<% if @hsponsors.size == 0 %>Be the first to become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a>
+<% elsif @hsponsors.size < 1 %>Become a <a href="/<%= @eventhome %>/sponsor/#sponsorship">Host Sponsor!</a><% end %>
```

Comment:
Mmmh, yes. I was a bit lazy, but one Host Sponsor is correct and it's working. If you don't mind I'd leave it as it is.